spk/bazarr/src/requirements-crossenv.txt

```diff
@@ -6,6 +6,7 @@
 
 greenlet==1.1.2
 numpy==1.22.0
+# lxml==4.6.3
```

Comment:
Not sure why it is commented?  FYI, I have left an example in `spk/python310/src/requirements-crossenv.txt` where `lxml` wheel depends on both `libxml2` and `libxslt` to build properly.
```
# [lxml]
# Depends: libxml2, libxslt
# lxml==4.8.0
```

Reply:
I had left it as a comment so that it doesn't get forgotten as the version against which it was tested.
But then it's sounds way simplier to maintain using the crossenv thingy if it does work.

Will be fixing this.

Reply:
Fixed using the example and it works very well, indeed.